src/ECharts/Theme.purs

```diff
@@ -0,0 +1,33 @@
+module ECharts.Theme
+  ( Theme(..)
+  , dark
+  , infographic
+  , macarons
+  , roma
+  , shine
+  , vintage
+  ) where
+
+import Data.Foreign (Foreign)
+
+foreign import forceExport :: Foreign
```

Comment:
One option would be to omit the foreign requires, and just tell library consumers to make sure the themes are included in their builds.